Formula/c/cucumber-cpp.rb

```diff
@@ -16,30 +17,32 @@ class CucumberCpp < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "nlohmann-json" => :build
+  # TODO: qt6 discovery not working, it finds 6.6.x but says it's unsuitable
```

Comment:
Not sure how important it is to have `qt` (6) also working for the formula? If it's not I can remove this comment.